src/components/Tables/MeetingVotesTableRow/MeetingVotesTableRow.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import DecisionResult from "../../Shared/DecisionResult";
+import { MATTER_STATUS_DECISION } from "../../../constants/ProjectConstants";
+import { IndividualMeetingVote } from "../../Shared/Types/IndividualMeetingVote";
+import { VOTE_DECISION } from "../../../constants/ProjectConstants";
+import { STYLES } from "../../../constants/StyleConstants";
+import { Link } from "react-router-dom";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+type MeetingVotesTableRowProps = {
+  /** the name of the matter that was voted on */
+  legislationName: string;
+  /** the index of the row */
+  index: number;
+  /** the voting body decision */
+  councilDecision: MATTER_STATUS_DECISION;
+  /** link to the detail page of the matter being voted on */
+  legislationLink: string;
+  /** date of the matter being voted on */
+  meetingDate: string;
+  /** vote results by individual members on the matter in this row */
+  votes: IndividualMeetingVote[];
+};
+
+function renderVotesCell(isExpanded: boolean, votes: IndividualMeetingVote[]) {
+  if (isExpanded) {
+    return (
+      <td>
+        {votes.map((vote, index) => {
+          return (
+            <div
+              key={`individual-vote-${index}-${vote.id}`}
+              style={{
+                display: "flex",
+                flexDirection: "row",
+                justifyContent: "center",
+                alignItems: "center",
+              }}
+            >
+              <DecisionResult result={vote.decision} />
+              <p style={{ flex: 1 }}>{vote.name}</p>
+            </div>
+          );
+        })}
+      </td>
+    );
+  } else {
+    let votesFor = 0;
+    let votesAgainst = 0;
+    let votesAbstained = 0;
+    votes.forEach((vote) => {
+      if (vote.decision === VOTE_DECISION.APPROVE) votesFor++;
+      if (vote.decision === VOTE_DECISION.REJECT) votesAgainst++;
+      if (vote.decision === VOTE_DECISION.ABSTAIN) votesAbstained++;
+    });
+    const votesMinified = `${votesFor} Approved   /   ${votesAgainst} Rejected   /   ${votesAbstained} Abstained`;
+    return (
+      <td>
+        <p>{votesMinified}</p>
+      </td>
+    );
+  }
+}
+
+const MeetingVotesTableRow = ({
+  index,
+  legislationLink,
+  legislationName,
+  councilDecision,
+  meetingDate,
+  votes,
+}: MeetingVotesTableRowProps) => {
+  const [expanded, setExpanded] = useState(false);
+
+  const backgroundColor = index % 2 === 0 ? STYLES.COLORS.EVEN_CELL : STYLES.COLORS.ODD_CELL;
+  const dateText = new Date(meetingDate).toDateString();
+  return (
+    <tr
+      style={{ backgroundColor }}
+      key={`voting-table-row-${index}`}
+      onClick={() => {
+        setExpanded(!expanded);
+      }}
+    >
+      <th scope="row">
+        <Link to={legislationLink}>{legislationName}</Link>
+      </th>
+      <td>
+        <p className="mzp-c-card-desc">{dateText}</p>
+      </td>
```

Comment:
Remove the date cell.

Any thoughts, @BrianL3 @JacksonMaxfield @hawkticehurst on whether the matter/legislation title should be together with the matter/legislation name or it should be on its own.

I think I prefer them to be together in the same cell.
Or if the title is too long of a description. We can make the legislation cell look like the legislation cell of the Voting Records table. Legislation name followed by some keywords.

Reply:
I put the legislation description underneath the name in the same cell.  Looks fine!